modules/creating-a-bias-metric-using-cli.adoc

```diff
@@ -106,4 +106,3 @@ The bias metrics request should return output similar to the following:
 ----
 
 The `specificDefinition` field is useful in understanding the real-world interpretation of these metric values. For this example, the model is quite fair over the `Is Male-Identifying?` field, with the rate of positive outcome only differing by approximately -0.3%.
```

Comment:
Not sure if this sentence is too long-winded: 

_is useful in understanding the real-world interpretation_

could it be cut to:

"...helps  you interpret..."?

Also what does "quite fair" mean? 

Why is just "quite" fair and not just "fair"?